infrastructure/k8s/helm-chart/eoapi/templates/stac/manifest.yaml

```diff
@@ -13,8 +13,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app: stac-{{ .Values.gitSha }}
-  name: stac-{{ .Values.gitSha }}
+    app: stac
+  name: stac
```

Comment:
we've tended to namespace the names per helm deployment, with something like https://github.com/IFRCGo/go-api/blob/terraform-prod/deploy/helm/ifrcgo-helm/templates/api/deployment.yaml#L5 , but actually I think I prefer without, as if we wanted multiple instances running in the same cluster, we could just put them in different kubernetes namespaces or so?

Would be curious to hear your thoughts though - I've been thinking about dropping that practice from our projects - it was something in some initial examples I worked with and have stuck with it, but I'm no longer sure if it's actually a good idea and tends to add a lot of verbosity to templates.

Reply:
Interesting, yeah given that you can apply a namespace at `helm install` time I don't see much reason to duplicate it in all the labels or names of resources.

I did notice that in the newer versions of k8s something has changed. I used to have my deploys without git sha (what you see above was a mistake). And I used to apply it to the pod label for the deploy selector.  Recently I've noticed that has become immutable and so you can't mutate via a new apply/patch anymore 🤷